src/OpenApi/src/Schemas/OpenApiJsonSchema.Helpers.cs

```diff
@@ -345,6 +345,17 @@ public static void ReadProperty(ref Utf8JsonReader reader, string propertyName,
                 schema.Metadata ??= new Dictionary<string, object>();
                 schema.Metadata[OpenApiConstants.RefId] = reader.GetString() ?? string.Empty;
                 break;
+            case OpenApiConstants.RefDescriptionAnnotation:
+                reader.Read();
+                schema.Metadata ??= new Dictionary<string, object>();
+                schema.Metadata[OpenApiConstants.RefDescriptionAnnotation] = reader.GetString() ?? string.Empty;
+                break;
+            case OpenApiConstants.RefExampleAnnotation:
```

Comment:
I don't think we're setting RefExampleAnnotation anywhere yet. I assume that is for a future PR?